ci/playbooks/content_provider/content_provider.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: Bootstrap step
+  ansible.builtin.import_playbook: "{{ cifmw_project_root }}/ci_framework/playbooks/01-bootstrap.yml"
+
+- hosts: "{{ cifmw_target_host | default('localhost') }}"
+  gather_facts: true
+  tasks:
+    - name: Install necessary dependencies
+      ansible.builtin.include_role:
+        name: 'install_yamls_makes'
+        tasks_from: 'make_download_tools'
+
+    - name: Deploy content provider registry
+      ansible.builtin.include_role:
+        name: registry_deploy
+
+    - name: Build Operators
+      ansible.builtin.include_role:
+        name: operator_build
+
+    - name: Get the containers list from container registry
+      ansible.builtin.command: curl -X GET localhost:5001/v2/_catalog
+      register: cp_imgs
+
+    - name: Add the container list to file
+      ansible.builtin.copy:
+        mode: 0644
+        content: "{{ cp_imgs.stdout }}"
+        dest: "{{ ansible_user_dir }}/local_registry.log"
```

Comment:
hmmm that path isn't collected, is it? We can iterate on that later, no biggies.

Reply:
We are collecting it, https://logserver.rdoproject.org/67/167/30999236d1ee09f544871a4065bc4ec772ef6b93/github-check/dataplane-operator-content-provider/db15184/local_registry.log